api/v2/Routes/students/index.js

```diff
@@ -21,8 +21,9 @@ router.use(RateLimit({
 router.use('/:email', validateAdminOrStudentMiddleware);
 
 // Ensure the requested student exists.
-router.use('/:email', async (req, _, next) => {
+router.use('/:email', async (req, res, next) => {
     const { email } = req.params;
+    res.locals.email = email
```

Comment:
What does this do?  I'm a bit confused.  Why can't we just get the email from the params?

Reply:
Using params didn't work on in the progressquerystring endpoint, it returned null.

Reply:
How do we test these endpoints on postman? What's an example get request

Reply:
Yeah I think it may be marked as id instead since we pass it along as id.  We can refactor to pass along as email if you want.  There is actually a ticket for that, so whatever is easier.  Good thing to do is to look at the query params and see what is being passed down.

Reply:
If we're going to refactor the params for students, wouldn't this change fall into that ticket?

Reply:
Yeah I guess we can do that.